install/updateVersion.sh

```diff
@@ -281,9 +281,10 @@ function update_istio_install() {
 
   execute_sed "s|{PILOT_HUB}|${PILOT_HUB}|" istio-sidecar-injector-configmap-debug.yaml.tmpl
   execute_sed "s|{PILOT_TAG}|${PILOT_TAG}|" istio-sidecar-injector-configmap-debug.yaml.tmpl
+  execute_sed "s|{PROXY_IMAGE}|${PROXY_IMAGE}|" istio-sidecar-injector-configmap-debug.yaml.tmpl
```

Comment:
debug and release templates can use `proxy` and `proxy_debug` image names directly. Otherwise we end up with a debug config that includes the release image or vice versa.

Reply:
https://github.com/istio/istio/pull/2902

Reply:
Sure that works.